Build fails on Fedora 28

When installing LTFS on an updated Fedora 28 server, build fails in the utils directory

Steps to reproduce the behavior:
- Install Fedora 28 and update (yum -y update), reboot to activate latest kernel
- Install prerequisite packages as described on homepage, plus net-snmp-devel
- Download ltfs head branch (git pull https://github.com/LinearTapeFileSystem/ltfs)
- Use default options to build (./autogen ; ./configure ; make)
- When in the utils directory, the build error below occurs:
make[3]: Entering directory '/root/ltfs/src/utils'
gcc -DHAVE_CONFIG_H -I. -I../..  -D_GNU_SOURCE -I../../src -DLTFS_CONFIG_FILE='"/usr/local/etc/ltASE_DIR='"NONE"' -I ..  -Wall -Wsign-compare -fsigned-char -I/usr/include/fuse -D_FILE_OFFSET_BITe/uuid -I/usr/include/libxml2 -I/usr/include -D ENABLE_SNMP -I. -I/usr/include -fkeep-inline-func -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -MT mkltfs-mkltfs.o -MD -MP -MF .deps/mkltfs-mkltfs.Tpo -c -o est -f 'mkltfs.c' || echo './'`mkltfs.c
mv -f .deps/mkltfs-mkltfs.Tpo .deps/mkltfs-mkltfs.Po
/bin/sh ../../libtool  --tag=CC   --mode=link gcc -Wall -Wsign-compare -fsigned-char -I/usr/incluFSET_BITS=64 -I/usr/include/uuid -I/usr/include/libxml2 -I/usr/include -D ENABLE_SNMP -I. -I/usr/ine-functions -rdynamic -g -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -Wl,--no-undefined,--as-needed -lfus  -lxml2   -L/usr/lib64 -licui18n -licuuc -licudata   -Wl,-z,relro -Wl,-z,now -L/usr/lib64 -lnets-ldl -lrpm -lrpmio -lnetsnmpagent -Wl,--enable-new-dtags -Wl,-z,relro -Wl,-z,now -specs=/usr/lib/hardened-ld -lnetsnmp -lssl -lssl -lcrypto -lm 2> /dev/null -Wl,-z,relro -Wl,-z,now -L/usr/lib64 ssl -lcrypto -lm -L../../messages -lbin_mkltfs_dat  -o mkltfs mkltfs-mkltfs.o ../libltfs/libltfs.
libtool: link: gcc -Wall -Wsign-compare -fsigned-char -I/usr/include/fuse -D_FILE_OFFSET_BITS=64 d -I/usr/include/libxml2 -I/usr/include -D ENABLE_SNMP -I. -I/usr/include -fkeep-inline-functions-D_FORTIFY_SOURCE=0 -O0 -ggdb -Wl,--no-undefined -Wl,--as-needed -pthread -Wl,-z -Wl,relro -Wl,-zble-new-dtags -Wl,-z -Wl,relro -Wl,-z -Wl,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -Wl,- -Wl,now -o .libs/mkltfs mkltfs-mkltfs.o  -L/usr/lib64 -L../../messages -lbin_mkltfs_dat ../liblto -lfuse -luuid -lxml2 -licui18n -licuuc -licudata -lnetsnmpmibs -lsensors -ldl -lrpm -lrpmio -lnnmp -lssl -lcrypto -lm -llibltfs_dat -linternal_error_dat -ltape_common_dat -lrt -lpthread -pthre/usr/local/lib
make[3]: *** [Makefile:466: mkltfs] Error 1
make[3]: Leaving directory '/root/ltfs/src/utils'
make[2]: *** [Makefile:682: all-recursive] Error 1
make[2]: Leaving directory '/root/ltfs/src'
make[1]: *** [Makefile:524: all-recursive] Error 1
make[1]: Leaving directory '/root/ltfs'
make: *** [Makefile:430: all] Error 2


 - OS: Fedora 28 Server (Linux vtape.consolitape.local 4.17.12-200.fc28.x86_64 #1 SMP Fri Aug 3 15:01:13 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux)

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -38,7 +38,7 @@ bin_PROGRAMS = ltfs
 
 ltfs_SOURCES = main.c ltfs_fuse.c
 ltfs_DEPENDENCIES = libltfs/libltfs.la ../messages/libbin_ltfs_dat.a
-ltfs_CPPFLAGS = @AM_CPPFLAGS@ -I ../ltfs-sde/src
+ltfs_CPPFLAGS = @AM_CPPFLAGS@ -I ../ltfs-sde/src -fPIC
 ltfs_LDADD = libltfs/libltfs.la
 ltfs_LDFLAGS = @AM_LDFLAGS@ -L../messages -lbin_ltfs_dat
 endif

--- a/src/utils/Makefile.am
+++ b/src/utils/Makefile.am
@@ -41,10 +41,10 @@ mkltfs_SOURCES = mkltfs.c
 mkltfs_DEPENDENCIES = ../libltfs/libltfs.la ../../messages/libbin_mkltfs_dat.a
 mkltfs_LDADD = ../libltfs/libltfs.la
 mkltfs_LDFLAGS = @AM_LDFLAGS@ -L../../messages -lbin_mkltfs_dat
-mkltfs_CPPFLAGS = @AM_CPPFLAGS@ -I ..
+mkltfs_CPPFLAGS = @AM_CPPFLAGS@ -I .. -fPIC
 
 ltfsck_SOURCES = ltfsck.c
 ltfsck_DEPENDENCIES = ../libltfs/libltfs.la ../../messages/libbin_ltfsck_dat.a
 ltfsck_LDADD = ../libltfs/libltfs.la
 ltfsck_LDFLAGS = @AM_LDFLAGS@ -L../../messages -lbin_ltfsck_dat
-ltfsck_CPPFLAGS = @AM_CPPFLAGS@ -I ..
+ltfsck_CPPFLAGS = @AM_CPPFLAGS@ -I .. -fPIC

